draft-ietf-rats-architecture.md

```diff
@@ -1110,8 +1116,15 @@ clocks between the Attester, Verifier, and Relying Party.
         |                 time(OP)                       |
 ~~~~
 
-The time considerations in this example are equivalent to those
-discussed under Example 1 above.
+In comparison with example 1, the time considerations in this example go into more detail with respect to the life-cycle of Claims and the requirements on time synchronization and corresponding timestamps.
+
+The goal is to create up-to-date and recent Evidence as soon as possible, in general. 
+
+At time(AA) the Attesting Environment is able to trigger an event (e.g. based on an Event-Condition-Action model) to create attestation Evidence as recent as possible. As soon as the Attesting Environment is aware, the new values are collected for composition into semantically enriched Claims at time Time(CC). Immediately after that, Evidence based on relevant "old" Claims and the most recent collected "new" Claims is generated at time(EG).
+
+In order to create attestation on demand due to the event trigger time(AA) at the time(EG), the Attester has be in synchronized state with respect to the Verifiers perception of global time. In order to accomplish his, the Attester has to have an identifier for time-bound recentness available at any time(HD) before time(EG) where the time interval between time(HD) and time(EG) is not too large due to time drift or other synchronization disablers (delta(time(HD),time(EG)<threshold)).
```

Comment:
time(AA) seems to be irrelevant for any security purpose, only time(EG) is relevant, so I don't think time(AA) belongs in this doc.

time(HD) also seems very confusing.   since time(HD) and time(EG) are on the same entity, and that same entity compares against a threshold, what security threat is this trying to address?  I don't follow.

Reply:
In order to always create the most recent Evdence possible, the Attester has to be aware of a changed Value as soon as possible. Otherwise Evidence is outdated and could even convey a misleading picture about Attester leading to incorrect risk assessmet. Hence time(AA) is quite relevant, I think. 

There are scenarios where this is less relevant (e.g. boot-time integrity) and some scenarios where this is more relevant (e.g. run-time integrity).

I agree that time(HD) is "squeezed" into the diagram it is actually the reception of the distributed handles. I did not want to add a "distributor of the handles" the the actors in the diagram. Maybe we can address this by rewording time(HD) correspondingly? In essence, I understand why this is difficult to follow at the moment and we have to fix this.

Reply:
Agreed that "HD" is not a good term, but a better term will be worked on.